.github/workflows/linux_meson.yml

```diff
@@ -121,6 +121,13 @@ jobs:
         export SCIPY_USE_PROPACK=1
         python dev.py --no-build test -j2 -- --durations 10 --timeout=60
 
+    - name: Run doctests
```

Comment:
It'd be preferable for this to replace the refguide check job here I think: https://github.com/scipy/scipy/blob/95cc0fd65011187c2589eaeddf0f0c86d04cc318/.circleci/config.yml#L170-L176. We don't need both.

Reply:
Maybe best move the refguide-check to GHA and combine with this one. Note that not all of refguide-check is obsoleted: the part which checks consistency between `__all__` lists and refguide is still needed (or if not, it's a separate discussion anyway).

Reply:
Ah, I didn't realize that we still need it for that, thanks.

> Maybe best move the refguide-check to GHA and combine with this one.

I think the point is that for docs-only PRs you can run with `[skip actions]` and trigger only the docs-related checks in CircleCI. If it's here, there's no good way to avoid running ~20 other jobs.